api/core/request_fields_test.go

```diff
@@ -270,6 +270,41 @@ func TestBadPostFilesStoreFail(test *testing.T) {
     }
 }
 
+func TestBadPostFilesFileSizeExceeded(test *testing.T) {
+    // Set size to 1 byte for testing. (a.txt is 2 bytes)
+    config.WEB_MAX_FILE_SIZE.Set(1);
```

Comment:
You will need to reset the value at the end of the test.
You should use a defer for that.